engine/menus/main_menu.asm

```diff
@@ -308,7 +310,14 @@ LinkCanceledText:
 
 StartNewGame:
 	ld hl, wd732
-	res 1, [hl]
+	; Ensure debug mode is not used when
+	; starting a regular new game.
+	; Bit 5 can be reset here as well to fix
+	; the issue documented in PrepareOakSpeech.
```

Comment:
Bug docs go in <https://github.com/pret/pokered/wiki/Bugs-and-Glitches>.
```suggestion
```

Reply:
(When/if they're relatively complete, and correct in what's actually completed, they can be added to the repo itself, with corresponding `BUG:` comments, but even more unsearchable comments like this don't need adding.)

Reply:
I would prefer starting a `docs/bugs_and_glitches.md` with a few fixes after this PR is merged. The wiki article has grown too big and a lot of the proposed fixes have not been adequately tested, including the one for this exact issue.

Reply:
If you have vetted fixes for some bugs on par with how the gen 2 docs are, then sure. A secondary goal is to get rid of the unsystematic comments throughout the source like "buggy", "broken", "this should be", "change this", etc.